airflow/providers/elasticsearch/log/es_task_handler.py

```diff
@@ -151,8 +151,10 @@ def __init__(
         self.formatter: logging.Formatter
         self.handler: logging.FileHandler | logging.StreamHandler  # type: ignore[assignment]
 
-    def _render_log_id(self, ti: TaskInstance, try_number: int) -> str:
+    def _render_log_id(self, ti: TaskInstance | TaskInstanceKey, try_number: int) -> str:
         with create_session() as session:
```

Comment:
probably we can / should avoid creating the session if not using USE_PER_RUN_LOG_ID